pkg/model/external_access.go

```diff
@@ -42,7 +42,7 @@ func (b *ExternalAccessModelBuilder) Build(c *fi.ModelBuilderContext) error {
 	}
 
 	// SSH is open to AdminCIDR set
-	if b.Cluster.IsTopologyPublic() {
+	if b.Cluster.Spec.Topology.Masters == kops.TopologyPublic {
 		for i, sshAccess := range b.Cluster.Spec.SSHAccess {
 			c.AddTask(&awstasks.SecurityGroupRule{
 				Name:          s("ssh-external-to-master-" + strconv.Itoa(i)),
```

Comment:
Here we are in `public` topology.. the security group rule is named `ssh-external-to-master-#`

Suggest:

```
Name:          s("ssh-"+ s(sshAccess) +"-to-master-" + strconv.Itoa(i)),
```

So that the name matches what the security group actually does